clients/python/pyproject.toml

```diff
@@ -135,3 +136,13 @@ show_column_numbers = true
 show_error_codes = true
 show_error_context = true
 ignore_missing_imports = true
+check_untyped_defs = true
```

Comment:
Without this, mypy will not check for type consistency in the bodies of functions that are not typed.

Reply:
sgtm